web_src/js/features/stopwatch.js

```diff
@@ -127,6 +127,7 @@ function updateStopwatchData(data) {
   const watch = data[0];
   const btnEl = $('.active-stopwatch-trigger');
   if (!watch) {
+    clearInterval(updateTimeInterval);
```

Comment:
```suggestion
      if (updateTimeInterval) {
        clearInterval(updateTimeInterval);
        updateTimeInterval = null;
      }
```

It should follow the line 118-121